pulpcore/cli/ansible/repository.py

```diff
@@ -150,13 +142,6 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
     The argument prefixed with the '@' can be the path to a JSON file with a list of objects."""
         ),
     ),
-    type_option(
-        choices={
-            "collection-version": PulpAnsibleCollectionVersionContext,
-            "role": PulpAnsibleRoleContext,
-        },
-        default="collection-version",
-    ),
```

Comment:
So these commands are supposed to handle all types of content in unison now?

Reply:
This option is now apart of the content command group. I've moved it back one spot in the command.